### PR TITLE
Add link to flag in notification message

### PR DESCRIPTION
**Pull request by dvik1950**
Solves "When a Flag is Submitted it Notifies Moderators on Discord with a Link to the Report in Active Admin." part of https://github.com/TheOdinProject/theodinproject/issues/1125.

### CODE REVIEW

File: spec/services/notifications/flag_submission_spec.rb [hunk 1]
```diff
@@ -2,21 +2,20 @@
 
 RSpec.describe Notifications::FlagSubmission do
   subject(:notification) do
-    described_class.new(
-      flagger: flagger_user,
-      project_submission: flagged_submission,
-      reason: 'I find it offensive'
-    )
+    described_class.new(flag)
```
Comment (KevinMulhern) [inline spec/services/notifications/flag_submission_spec.rb:5]: can this subject all go on one line now?

**Review by KevinMulhern — CHANGES_REQUESTED** [review]
Looks really good @dvik1950, just a few suggestions :)

File: app/controllers/project_submissions/flags_controller.rb [hunk 2]
```diff
@@ -30,11 +26,7 @@ def notify_discord_admins
     return if Rails.env.development?
 
     DiscordNotifier.notify(
-      Notifications::FlagSubmission.new(
-        flagger: current_user,
-        project_submission: project_submission,
-        reason: params[:reason]
-      )
+      Notifications::FlagSubmission.new(@flag)
```
[the remark this answers is not in the document]
Comment (dvik1950) [reply app/controllers/project_submissions/flags_controller.rb]: Thanks for the detailed review and sorry for the delay. I made some changes, I'm not sure if I'm on the right track with the changes made regarding this comment though. The comment makes perfect sense but I'm not sure this is the correct way to achieve what you mean 😬

[the remark this answers is not in the document]
Comment (KevinMulhern) [reply app/controllers/project_submissions/flags_controller.rb]: What you've done is exactly what I meant @dvik1950 😄

**Comment by KevinMulhern** [comment]
This is great, thanks @dvik1950 🚀

**Review by KevinMulhern — APPROVED** [review]

File: app/controllers/project_submissions/flags_controller.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (dvik1950) [reply app/controllers/project_submissions/flags_controller.rb]: glad to hear that 😬

**Pull request merged**